pkg/imgpkg/cmd/copy_repo_src.go

```diff
@@ -77,34 +82,41 @@ func (c CopyRepoSrc) CopyToRepo(repo string) (*ctlimgset.ProcessedImages, error)
 		return nil, fmt.Errorf("Building import repository ref: %s", err)
 	}
 
+	c.logger.Debugf("copy the fetched images\n")
 	processedImages, ids, err := c.imageSet.Relocate(unprocessedImageRefs, importRepo, c.registry)
 	if err != nil {
 		return nil, err
 	}
 
+	for _, bundle := range bundles {
```

Comment:
shouldnt bundles also contain `originBundle` so that it can get a NoteCopy call?

Reply:
the originBundle is added as part of the call to `Bundle.AllImagesRefs`